controllers/userController.js

```diff
@@ -21,4 +22,21 @@ const userRegister = async (req, res) => {
   }
 }
 
-module.exports = { userRegister }
+const login = async (req, res) => {
+  const { username, password } = req.body;
+  try {
+    const dbUser = await userModel.findOne({ where: { username: username } })
```

Comment:
Put a condition if user is not present then pass error not move the next function.